tutorials/experiments/langchain_email_extraction.ipynb

```diff
@@ -1,269 +1,2445 @@
 {
```

Comment:
<p>Let's add banner and some links at the top.</p><p> </p><p><a href="https://github.com/Arize-ai/phoenix/blob/main/tutorials/experiments/datasets_and_experiments_quickstart.ipynb" target="_blank">https://github.com/Arize-ai/phoenix/blob/main/tutorials/experiments/datasets_and_experiments_quickstart.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/3805/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>